controllers/tests/kafkacluster_controller_externalnodeport_test.go

```diff
@@ -237,7 +237,7 @@ var _ = Describe("KafkaClusterNodeportExternalAccess", func() {
 						ContainerPort: 9733,
 						Type:          "plaintext",
 					},
-					ExternalStartingPort: 30100,
+					ExternalStartingPort: 30300,
```

Comment:
I had to change this to 30300 as 30100 seems to be already used in the new version of envtest